src/ontology/mondo-edit.obo

```diff
@@ -388432,7 +388435,7 @@ synonym: "hypospadias-hypertelorism syndrome" EXACT [Orphanet:2745]
 synonym: "Opitz BBBG syndrome" RELATED [GARD:0000193]
 synonym: "Opitz G syndrome" EXACT [NCIT:C125487]
 synonym: "Opitz G/BBB syndrome" EXACT [DOID:0050780]
-synonym: "Opitz GBBB syndrome" EXACT [OMIMPS:300000]
+synonym: "Opitz GBBB syndrome" EXACT [OMIM:300000]
```

Comment:
@twhetzel I don't understand why this x-ref was updated to OMIM:300000, since OMIM:300000 refers to another Mondo ID. 
I think this OMIMPS x-ref should be kept or another x-ref should be used for this synonym (but not OMIM:300000), but I need to know more before saying that this is what should be done.

Reply:
https://omim.org/phenotypicSeries/PS300000 is no longer active. what xref for this synonym should be used now?

Reply:
the xref should be any source(s) that use this synonym. It could be DO, Orphanet,etc., a PMID, and/or your orcid.

Reply:
Ah, ok I think I understand now what you are asking so I've updated the synonym xref to sources that use that term.